views/home.ejs

```diff
@@ -1 +1,31 @@
-<h1> <%= title %> Home Page</h1>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
Will bootstrap work without meta utf-8 and viewport tags??

Reply:
Added meta tags in layout.ejs files. Thanks.